docs/api-reference.md

```diff
@@ -77,22 +78,33 @@ returned at the time.
 
 ### `rerender`
 
-```js
-function rerender(newProps?: any): void
+```ts
+rerender: (newProps?: any) => void
 ```
 
 A function to rerender the test component, causing any hooks to be recalculated. If `newProps` are
 passed, they will replace the `callback` function's `initialProps` for subsequent rerenders.
 
 ### `unmount`
 
-```js
-function unmount(): void
+```ts
+unmount: () => void
 ```
 
 A function to unmount the test component. This is commonly used to trigger cleanup effects for
 `useEffect` hooks.
 
+### `hydrate`
```

Comment:
I know you've got the message about this only being available in the server renderer, but I wonder if a seperate section for SSR would make it even more apparent?